v2/yarpcjson/register.go

```diff
@@ -34,45 +34,30 @@ var (
 	_interfaceEmptyType = reflect.TypeOf((*interface{})(nil)).Elem()
 )
 
-// Procedure builds a TransportProcedure from the given JSON handler. handler must be
+// Procedure builds an EncodingProcedure from the given JSON handler. handler must be
 // a function with a signature similar to,
 //
 // 	f(ctx context.Context, body $reqBody) ($resBody, error)
 //
-// Where $reqBody and $resBody are a map[string]interface{} or pointers to
-// structs.
-func Procedure(name string, handler interface{}) []yarpc.TransportProcedure {
-	return []yarpc.TransportProcedure{
+// Where $reqBody and $resBody are of type map[string]interface{}, interface{}, or
+// struct pointers.
+func Procedure(name string, handler interface{}) []yarpc.EncodingProcedure {
```

Comment:
Again, not actionable here since it looks like this was a relic of the past, but if we always just return a single `EncodingProcedure` in the slice, it might make more sense to update the signature to be a single procedure.  If this is largely for convenience, why not name it `Procedures`?